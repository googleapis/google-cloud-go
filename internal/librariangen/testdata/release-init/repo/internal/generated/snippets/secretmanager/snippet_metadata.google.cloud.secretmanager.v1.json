{
  "clientLibrary": {
    "name": "cloud.google.com/go/secretmanager",
    "version": "$VERSION",
    "language": "GO",
    "apis": [
      {
        "id": "google.cloud.secretmanager.v1",
        "version": "v1"
      }
    ]
  }
}
