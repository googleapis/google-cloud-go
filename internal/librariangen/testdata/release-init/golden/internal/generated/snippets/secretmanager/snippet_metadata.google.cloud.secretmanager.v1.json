{
  "clientLibrary": {
    "name": "cloud.google.com/go/secretmanager/apiv1",
    "version": "1.16.0",
    "language": "GO",
    "apis": [
      {
        "id": "google.cloud.secretmanager.v1",
        "version": "v1"
      }
    ]
  },
  "snippets": []
}
